src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int numberOfBalls = 3;
+        for (int i = 0; i < numberOfBalls; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
Please do not call `.toString()` inside `System.out.println()`, because it' called **under the hood** inside this method.